onchain/rollups/package.json

```diff
@@ -24,7 +24,7 @@
         "info": "npm-scripts-info",
         "proofs:update": "./test/foundry/dapp/helper/update-proofs.sh",
         "proofs:setup": "./test/foundry/dapp/helper/update-proofs.sh --setup",
-        "prepack": "run-s build tsc:prod copy-dts",
+        "prestart": "run-s build tsc:prod copy-dts",
```

Comment:
I don't understand this change.

Reply:
You're right, `prestart` is not neither defining better nor is a better use in yarn pnp. In fact, all `Pre` and `post` scripts have been deliberately deprecated, we should avoid them in a near future. We can tackle this in the next iteration.